lib/nsxv.py

```diff
@@ -614,10 +616,11 @@ def __str__(self):
         term_str = str(term)
         if term_str:
           target.append(term_str)
-      target.append('\n')
 
       # ensure that the header is always first
       target = target_header + target
       target.append('%s' % (_XML_TABLE.get('sectionEnd')))
       target.append('\n')
-    return '\n'.join(target)
+
+      target_as_xml = xml.dom.minidom.parseString(''.join(target))
+    return target_as_xml.toprettyxml(indent=" ")
```

Comment:
I would prefer that the header matches that NSX does when it gets extracted. So if NSX adds the encoding lets get it added. If this currently matches the NSX output we can leave it.